content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md

```diff
@@ -0,0 +1,202 @@
+
+During the course of the year, we’re looking forward to graduate the schema to General Availability `v1`.` If you’re interested in this effort, you can also join [WG Component Standard](https://github.com/kubernetes/community/tree/master/wg-component-standard).
+
+## What’s next?
+
+### 2019 plans
+
+We are focusing our efforts this year around graduating the configuration file format to GA (`kubeadm.k8s.io/v1`)`, graduating this super-easy High Availability flow to stable, and providing better tools around rotating certificates needed for running the cluster automatically.
+
+In addition to these three key milestones of our charter, we want to improve the following areas:
+
+*   Support joining Windows nodes to a kubeadm cluster (with end-to-end tests)
+*   Improve the upstream CI signal, mainly for HA and upgrades
+*   Consolidate how Kubernetes artifacts are built and installed
+*   Utilize Kustomize to allow for advanced, layered and declarative configuration
+
+We make no guarantees that these deliverables will ship this year though, as this is a community effort. If you want to see these things happen, please join our SIG and start contributing! The ComponentConfig issues in particular need more attention.
+
+
+### kubeadm now has a logo!
+
+[Dan Kohn](https://github.com/dankohn) offered CNCF’s help with creating a logo for kubeadm in this cycle. [Alex Contini](https://github.com/alexcontini) created 19 (!) different logo options for the community to vote on. The public poll was active for around a week, and we got 386 answers. The winning option got 17.4% of the votes, so now it’s official!
+
+![kubeadm's logo](/static/images/blog/2019-06-24-kubeadm-ha-v115/logo.png)
+
+## Contributing
+
+If this all sounds exciting, **join us**!
+
+[SIG Cluster Lifecycle](https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle) has many different
+subprojects, where kubeadm is one of them. In the following picture you can see that there are many pieces in the
+puzzle, and we have a lot still to do.
+
+![SIG Cluster Lifecycle Projects](/static/images/blog/2019-06-24-kubeadm-ha-v115/projects.png)
+
+Some handy links if you want to start contribute:
+
+*   We have a recorded [New Contributor Onboarding](https://www.youtube.com/watch?v=Bof9aveB3rA) session on YouTube.
+*   Look out for “good first issue”, “help wanted” and “sig/cluster-lifecycle” labeled issues in our repositories 
+    (e.g. [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm))
+*   Join **#sig-cluster-lifecycle**, **#kubeadm**, **#cluster-api**, **#minikube**, **#kind**, etc. in Slack
```

Comment:
Which Slack organisation? (I assume it's the main Kubernetes Slack).
Is it worth linking there?